tests/unit/test-safe-svg-tags.php

```diff
@@ -47,4 +47,14 @@ public function test_get_tags() {
 		$this->assertContains( 'customTag', $svg_tags );
 		$this->assertSame( $svg_tags, $filtered_svg_tags );
 	}
+
+	/**
+	 * Test the kses_allowed_html function.
+	 *
+	 * @throws PHPUnit\Framework\AssertionFailedError If the function does not return an array.
+	 */
+	public function test_kses_allowed_html() {
+		$allowed_html = SafeSvg\SafeSvgTags\safe_svg_tags::kses_allowed_html();
+		$this->assertIsArray( $allowed_html );
```

Comment:
**thought (non-blocking):** This feels like adding a test just to have a test. I'd prefer it if we were either verifying that the correct items were stripped from an SVG, or that the correct params/attributes are included.